tests/lib/resource_test.py

```diff
@@ -55,7 +56,7 @@
 
 @pytest.fixture
 def resource():
-    return Resource(TYPE_NAME, None)
+    return Resource(TYPE_NAME, None, None)
```

Comment:
I am guessing the None is for typeConfig. Can we remove it from here to make sure TypeConfig is not made a required value in the future